packages/EasyErrorHandler/src/Bridge/Symfony/Commands/AnalyzeErrorCodesCommand.php

```diff
@@ -0,0 +1,126 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EonX\EasyErrorHandler\Bridge\Symfony\Commands;
+
+use EonX\EasyErrorHandler\Bridge\Symfony\Interfaces\ErrorCodes\ErrorCodesProviderInterface;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Helper\Table;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+final class AnalyzeErrorCodesCommand extends Command
+{
+    /**
+     * @var int
```

Comment:
As I know @natepage agreed that we can omit annotations for constants.

Reply:
Refactored.

Reply:
Why did you delete this constant?

Reply:
@itorgov We use the value only once, I think that using a constant to store it is unnecessary.

Reply:
I know but usually we use constats in such cases even there is only one place where we use the value.
@akl-95 wdyt?

Reply:
This const is stull useful I believe since it makes the magic value `100` clear.